### PR TITLE
Add option to hide an empty listing block.

### PR DESCRIPTION
**Pull request by mbaechtold**
This option makes use of a feature in "ftw.simplelayout" where a block can be hidden visually.

### CODE REVIEW

File: ftw/events/browser/eventlistingblock.py [hunk 1]
```diff
@@ -16,6 +17,31 @@
...
+    def __call__(self, *args, **kwargs):
+        if IHiddenBlock(self.context, False):
+            raise Exception('You are not allowed to add the IHiddenBlock '
+                            'behavior on the event listing block.')
+
+        # Get the items. This can be used from the template too.
+        self.items = self.get_items()
+
+        # Set an attribute which will be checked by "ftw.simplelayout".
+        setattr(
```
Comment (maethu) [inline ftw/events/browser/eventlistingblock.py]: @mbaechtold This is not a good behavior `setting something on access`.  I personally do not expect a database change if I vieu a block.

Comment (maethu) [inline ftw/events/browser/eventlistingblock.py]: This should somehow happen in `contents/eventlistingblock.py`

Comment (mbaechtold) [inline ftw/events/browser/eventlistingblock.py]: @maethu, I've modified the implementation which includes some hints I got from @jone.

**Comment by mbaechtold** [comment]
@maethu, could you please have another look?

**Pull request merged**